eng/automation/LocProject.json

```diff
@@ -4,10 +4,10 @@
         "LanguageSet": "VS_Main_Languages",
         "LocItems": [
             {
-                "SourceFile": "src\\Compatibility\\Core\\src\\iOS\\Resources\\StringResources.resx",
-                "LclFile": "loc\\{Lang}\\src\\Compatibility\\Core\\src\\iOS\\Resources\\StringResources.resx.lcl",
+                "SourceFile": "src\\Controls\\src\\Core\\src\\Compatibility\\iOS\\Resources\\StringResources.resx",
+                "LclFile": "loc\\{Lang}\\src\\Controls\\src\\Core\\src\\Compatibility\\iOS\\Resources\\StringResources.resx.lcl",
                 "CopyOption": "LangIDOnName",
-                "OutputPath": "src\\Compatibility\\Core\\src\\iOS\\Resources\\"
+                "OutputPath": "src\\Controls\\src\\Core\\src\\Compatibility\\iOS\\Resources\\xlf\\"
```

Comment:
I think this broke in https://github.com/dotnet/maui/commit/5e8e41a691d0c8b8ca759c182e6ae89ea3d5c3a3#diff-53f6fc5bae2a47267d1dca158becf5daebd643a36298ff93b40e637d77168ced

These things are not run as part of PRs, so they got missed and not checked.